Tests/ParseSwiftTests/ParseObjectCommandTests.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  ParseObjectCommandTests.swift
+//  ParseSwiftTests
+//
+//  Created by Corey Baker on 7/19/20.
+//  Copyright © 2020 Parse. All rights reserved.
+//
+
+import Foundation
+import XCTest
+@testable import ParseSwift
+
+class ParseObjectCommandTests: XCTestCase {
+
+    struct GameScore: ParseSwift.ObjectType {
+        //: Those are required for Object
+        var objectId: String?
+        var createdAt: Date?
+        var updatedAt: Date?
+        var ACL: ACL?
+
+        //: Your own properties
+        var score: Int
+
+        //: a custom initializer
+        init(score: Int) {
+            self.score = score
+        }
+    }
+
+    let parseDateEncodingStrategy: ParseEncoder.DateEncodingStrategy = .custom({ (date, enc) in
```

Comment:
Does this have to be redefined here? Is it possible to use the one that's in `ParseCoding`? You might have to pull my PR into your branch for those changes to become visible, or we could maybe change this after merging?

Reply:
This was left over from when I was testing the JSON decoder and date. I'll remove as I'm not using it